content/applications/general/iot/config/updating_iot.rst

```diff
@@ -73,6 +73,100 @@ Lastly, click :guilabel:`Flash`, and wait for the process to finish.
    An alternative software for flashing the micro SD card is `Raspberry Pi Imager
    <https://www.raspberrypi.com/software/>`_.
 
+Windows IoT update
+==================
+
+Windows virtual :abbr:`IoT (Internet of Things)` box may occasionally need an update to work
+properly.
+
+The following processes cover the :ref:`uninstallation <iot/config/uninstall-windows-iot>` and
+:ref:`re-installation <iot/config/re-install-windows-iot>` of the Windows virtual :abbr:`IoT
+(Internet of Things)` box.
+
+.. _iot/config/uninstall-windows-iot:
+
+Uninstalling Windows IoT
+------------------------
+
+Prior to upgrading the Windows virtual :abbr:`IoT (Internet of Things)` box, the previous version
+should be uninstalled first.
+
+.. important::
+   Before uninstalling a previous version of the Windows virtual :abbr:`IoT (Internet of Things)`
+   box, ensure that there is a newer version of Odoo's virtual :abbr:`IoT (Internet of Things)` box
```

Comment:
this is referred to throughout as _Windows virtual IoT box_, so just changing to be consistent
```suggestion
   box, ensure that there is a newer version of Windows virtual :abbr:`IoT (Internet of Things)` box
```